.prettierignore

```diff
@@ -38,9 +38,6 @@ next-app/**/.env.*
 
 ## Extras
 # Libraries and generated code
-src/help/template/scripts/
```

Comment:
yes, I'm fine removing this help directory - it was shipped for a while, but we couldn't update it directly and so eventually we removed references to it.